ocaml-lsp-server/src/custom_requests/req_typed_holes.ml

```diff
@@ -0,0 +1,48 @@
+open Import
+
+let capability = ("handleTypedHoles", `Bool true)
+
+let meth = "ocamllsp/typedHoles"
+
+let failwith_err err = failwith @@ Printf.sprintf "%s: %s" meth err
```

Comment:
We have two kinds of errors, differing by what is their ultimate cause:

* When the cause is a programming error on the server side, we should raise a code error.
* When the cause is a misbehaving client, we should raise `Response.Error.E`.

Therefore, we should never use failwith.